packages/driver/src/dom/elements/find.ts

```diff
@@ -289,3 +289,11 @@ export const getContainsSelector = (text, filter = '', options: {
 
   return selectors.join()
 }
+
+export const getInputFromLabel = ($el) => {
```

Comment:
Does this support the case where an input is nested inside a label but the input doesn't have an id?

Reply:
Oh, I forgot that would work. Fixing and adding a test for it.